src/net/chat.rs

```diff
@@ -701,8 +686,10 @@ impl ConversationP2P {
         let res = self.connection.send_data(w);
         match res {
             Ok(num_sent) => {
-                self.stats.last_send_time = get_epoch_time_secs();
-                self.stats.bytes_tx += num_sent as u64;
+                if num_sent > 0 {
+                    self.stats.last_send_time = get_epoch_time_secs();
+                    self.stats.bytes_tx += num_sent as u64;
```

Comment:
Should we use some safe math here?

Reply:
Up :)

Reply:
I don't think it's necessary.  The conversation statistics get reset whenever the socket is dropped or reconnected, which means that this can overflow only if we successfully send 2**64 - 1 bytes without interruption (highly unlikely -- even _counting_ this high would take a long time).